src/components/privateWrapper/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+export const PrivateWrapperComponent = ({ isAuthenticated, children }) => (
+    isAuthenticated && <span>{ children }</ span>
+);
+
+const mapStateToProps = state => ({
+  isAuthenticated: !!state.account.publicKey,
```

Comment:
this should be something else then `publicKey` because now it shows the haeder right when I click the "Login" button, but it should only after the server request is finished and it is redirected from login page.